instrumentation/src/javatests/org/oppia/android/instrumentation/application/EndToEndTestImageParsingModuleTest.kt

```diff
@@ -0,0 +1,103 @@
+package org.oppia.android.instrumentation.application
+
+import android.app.Application
+import android.content.Context
+import androidx.test.core.app.ApplicationProvider
+import androidx.test.ext.junit.runners.AndroidJUnit4
+import com.google.common.truth.Truth.assertThat
+import dagger.BindsInstance
+import dagger.Component
+import dagger.Module
+import dagger.Provides
+import org.junit.Before
+import org.junit.Test
+import org.junit.runner.RunWith
+import org.oppia.android.util.parser.image.DefaultGcsPrefix
+import org.oppia.android.util.parser.image.ImageDownloadUrlTemplate
+import org.oppia.android.util.parser.image.ThumbnailDownloadUrlTemplate
+import org.robolectric.annotation.Config
+import org.robolectric.annotation.LooperMode
+import javax.inject.Inject
+import javax.inject.Singleton
+
+@RunWith(AndroidJUnit4::class)
+@LooperMode(LooperMode.Mode.PAUSED)
+@Config(application = EndToEndTestImageParsingModuleTest.TestApplication::class)
+class EndToEndTestImageParsingModuleTest {
```

Comment:
Adding these tests is problematic since they'll be ignored by your filter in ComputeAffectedTests. We ought to actually filter for instrumentation tests, but in the meantime could you filter out just the player test instead of all of instrumentation/?

Reply:
Done

Reply:
Just looking to validate that they're running now in CI (see my comment in #3505).